packaging/makeself/jobs/10-prepare-destination.install.sh

```diff
@@ -4,18 +4,20 @@
 # shellcheck source=packaging/makeself/functions.sh
 . "$(dirname "${0}")/../functions.sh" "${@}" || exit 1
 
-[ -n "${GITHUB_ACTIONS}" ] && echo "::group::Preparing build environment"
+# shellcheck disable=SC2015
+[ "${GITHUB_ACTIONS}" = "true" ] && echo "::group::Preparing build environment" || true
```

Comment:
FWIW, I find these messages helpful even when building locally.

Reply:
Hmm, good point. It would make some sense to update them in a way that is easily searchable for local builds.